lib/resources/apache_conf.rb

```diff
@@ -1,14 +1,13 @@
 # encoding: utf-8
 # copyright: 2015, Vulcano Security GmbH
-# author: Dominik Richter
-# author: Christoph Hartmann
 
 require 'utils/simpleconfig'
 require 'utils/find_files'
 
 module Inspec::Resources
   class ApacheConf < Inspec.resource(1)
     name 'apache_conf'
+    supports platform: 'unix'
```

Comment:
Since we already have os_family here defined lets update it to be:
```
supports platform: 'linux'
supports platform: 'debian'
```
instead of unix, and remove the os_family's

Reply:
alright